src/chunkserver/chunk_service.cpp

```diff
@@ -65,8 +66,15 @@ void ChunkServiceImpl::DeleteChunk(RpcController *controller,
                                                response,
                                                done);
     CHECK(nullptr != closure) << "new chunk service closure failed";
-
     brpc::ClosureGuard doneGuard(closure);
+    // auth
+    if (!Authenticator::GetInstance().VerifyCredential(
+        request->authtoken())) {
+        LOG(ERROR) << "DeleteChunk auth failed, request = "
+                   << request->ShortDebugString();
```

Comment:
Is token is a visable string if we logging it?

Reply:
no token is encrypted data